### PR TITLE
Import lrzsz from oldpackages

### PR DESCRIPTION
**Pull request by kuoruan**
Signed-off-by: Hsing-Wang Liao <kuoruan@gmail.com>

### CODE REVIEW

**Comment by hnyman** [comment]
The commit message is missing signed-off-by line. Please add that.

You also need to declare yourself as the PKG_MAINTAINER.
Additionally, check the package's PKG_LICENSE field etc. missing info.
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

File: utils/lrzsz/Makefile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+PKG_MD5SUM:=b5ce6a74abc9b9eb2af94dffdfd372a4
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/lrzsz
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=X,Y and Z-modem protocols
```
Comment (mhei) [inline utils/lrzsz/Makefile]: Please. one whitespace after semicolon, i.e. between "X, Y ..."

File: utils/lrzsz/Makefile [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+endef
+
+define Package/lrzsz/description
+	Transfer files in your login sessions.
+	Very leightweight and straight forward.
+	You just need a terminal client that can do
+	either X, Y or Z-modem file transfers.
+endef
+
+# uses GNU configure
```
Comment (mhei) [inline utils/lrzsz/Makefile]: I think we don't need this line.

**Comment by kuoruan** [comment]
OK, fixed. Please check it again.

**Comment by yousong** [comment]
Just tried sz, rz this on malta with ssh in iTerm2 and it works great!  If there is no further comments on this one, I'd like to merge this a while later.

**Comment by kuoruan** [comment]
Got it.

**Comment by yousong** [comment]
thanks

**Pull request merged**